robot/EDA/resources/TriggerHandlersPageObject.py

```diff
@@ -0,0 +1,195 @@
+            trigger_handlers_locators["recently_viewed"],
+            error=locator_trigger + " locator is not available in Trigger Handlers page"
+        )
+        self.selenium.click_element(trigger_handlers_locators["recently_viewed"])
+
+        self.selenium.wait_until_page_contains_element(
+            trigger_handlers_locators["all_main_fields"],
+            error=trigger_handlers_locators["all_main_fields"] + " locator is not available in Trigger Handlers page"
+        )
+        self.selenium.click_element(trigger_handlers_locators["all_main_fields"])
+
+        self.selenium.wait_until_page_contains_element(
+            trigger_handlers_locators["search_exists"],
+            error=trigger + " object not found in Trigger Handlers"
+        )
+        self.selenium.click_element(trigger_handlers_locators["search_exists"])
+        self.selenium.get_webelement(trigger_handlers_locators["search_exists"]).send_keys(trigger + Keys.RETURN)
+
+        # Verify that the CON_PreferredPhone_TDTM object has been found
+        self.selenium.wait_until_page_contains_element(
+            trigger_handlers_locators["preferred_phone"],
+            error="CON_PreferredPhone_TDTM not found in Trigger Handler list"
+        )
+
+        time.sleep(2)
+        self.selenium.wait_until_page_contains_element(
+            trigger_handlers_locators["trigger_handler"],
+            error="Unable to find Preferred Phone Trigger Handler "
+        )
+        self.selenium.click_element(trigger_handlers_locators["trigger_handler"])
+
+    def select_active_checkbox(self, loc_check, loc_checkbox):
+        """ Selects checkbox.  Does nothing if checkbox is already checked """
+        if self.check_if_element_exists(loc_check):
+            return
+        else:
+            self.selenium.click_button("Edit Active")
+            self.selenium.get_webelement(loc_checkbox).click()
+            self.selenium.click_button("Save")
+            self.selenium.wait_until_element_is_visible(loc_check)
+            return
+
+    def set_trigger_to_active(self, trigger):
+        """ Set Preferred Phone trigger to Active.  Leave Active if already checked """
+
+        if self.check_if_element_exists(trigger_handlers_locators["checked_and_active"]) or self.check_if_element_exists(trigger_handlers_locators["trigger_active_check"]):
+            return
+        else:
+            self.selenium.click_button("Edit Active")
+            self.click_item(
+                trigger_handlers_locators["edit_mode_unchecked"], 
+                "cannot find active checkbox", 
+                True
+            )
+            self.selenium.click_element(trigger_handlers_locators["trigger_save"])
+            time.sleep(1)
+            #self.selenium.wait_until_element_is_visible(trigger_handlers_locators["trigger_active_check"])
+            return
+
+    def Clear_the_check_on_active_checkbox(self):
+        """ Clear the Preferred Phone trigger """
```

Comment:
The documentation and the name of the keyword seem to say two different things, though I don't understand all of the EDA terminology here. Is "clearing the preferred phone trigger" the same thing as "clearing the check on active checkbox"?

Reply:
resolved

Reply:
I removed this file.